pkg/local_object_storage/shard/writecache.go

```diff
@@ -0,0 +1,36 @@
+package shard
+
+import (
+	"errors"
+
+	"github.com/nspcc-dev/neofs-node/pkg/local_object_storage/shard/mode"
+)
+
+// ErrWriteCacheDisabled is returned when an operation on write-cache is performed,
+// but write-cache is disabled.
+var ErrWriteCacheDisabled = errors.New("write-cache is disabled")
```

Comment:
Can we leave it unexported? I don't see where else we use it. Same for `ErrMustBeReadOnly`.

Reply:
Fixed